config/config-example.yaml

```diff
@@ -4,27 +4,236 @@ receivers:
     collection_interval: 1m
     scrapers:
       memory:
+      cpu:
+      disk:
+      filesystem:
+      network:
+      swap:
+      process:
```

Comment:
None of the `process` metrics will work unless you use the custom processor that's pending the other PR: https://github.com/GoogleCloudPlatform/opentelemetry-operations-
collector/pull/2 (similar for a few of the other metrics)

I can chase up @nilebox to review that if this is a blocker for you

Reply:
This is merged now if you want to update your config

Reply:
I did update the config with some re-orderings. Is that what you are referring to? The `process` metric is already in the config.

Reply:
Some of the metrics you currently have configured to be scraped won't work unless I'm missing something, e.g.

- You have the `process` scraper configured. The OT process metrics store process related attributes in the resource object whereas Cloud Monitoring stores these as labels (https://cloud.google.com/monitoring/api/metrics_agent#agent-processes). This requires a fairly complex transformation not supported by the metrics transform processor so you need to add the `agentmetrics` processor to do that
- Likewise that processor takes care of splitting some metrics (disk_read, disk_write) & computing utilizations